test/Pester/PlatyPs.Tests.ps1

```diff
@@ -262,7 +262,7 @@ Describe 'New-MarkdownHelp' {
         }
 
         It 'generates markdown with correct synopsis' {
-            ($content | Where-Object {$_ -eq 'Adds a file name extension to a supplied name.'} | Measure-Object).Count | Should Be 1
+            ($content | Where-Object {$_ -eq 'Adds a file name extension to a supplied name.'} | Measure-Object).Count | Should Be 2
```

Comment:
where that 2 comes from? :)

Reply:
@vors One from the syntax and the other from the description. It also has the same string.

Reply:
Hmm.. in CI the output seems to be 1 as expected before the change. When running on my machine, it comes to be 2.

Reply:
oh, so you should run `Update-Help` locally, like in https://github.com/PowerShell/platyPS/blob/92842544ca2482d55964e0ba109af6b724c2215c/appveyor.yml#L18